aten/src/ATen/native/native_functions.yaml

```diff
@@ -78,7 +78,6 @@
   use_c10_dispatcher: full
   manual_kernel_registration: True
   variants: method
-  supports_named_tensor: True
```

Comment:
Could you also delete https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/README.md#supports_named_tensor ?